db_json_util.py

```diff
@@ -59,62 +59,69 @@ def main() -> None:
-                    else:
-                        print('exiting, no data exported')
-                    return
-                else:
-                    print('error occured, exiting')
+            api_export_to_json_files(path_chains, path_urls, args.url, args.auth_password)
 
     if args.local:
         if args.import_data:
             local_import_from_json_files(path_chains, path_urls, path_db_file)
         if args.export_data:
             local_export_to_json_files(path_chains, path_urls, path_db_file)
 
+def api_export_to_json_files(json_chains: Path, json_rpc_urls: Path, api_url: str, auth_pw: str = ""):
+    """
+    Exports data from the SQLite database to JSON files.
+    Assumes the JSON files has a specific format as defined by XYZ.
+    """
+
+    if json_chains:
+        response = requests.get(api_url + '/all/chains')
+        if response.status_code == 200:
+            data_chains = response.json()
+        else:
+            print(response.text)
+            return
+        # check if path exists
+        if not json_chains.exists():
+            os.makedirs(os.path.dirname(json_chains), exist_ok=True)
+            with open(json_chains, 'w', encoding='utf-8') as f:
+                json.dump(data_chains, f, indent=4)
+                print(f'exported chains to {json_chains}')
+        elif json_chains.exists():
+            user_input = input("File already exists for json chain, overwrite? (y/n): ")
+            if user_input in ['y', 'Y', 'yes', 'Yes', 'YES']:
+                with open(json_chains, 'w', encoding='utf-8') as f:
+                    json.dump(data_chains, f, indent=4)
+                    print(f'exported chains to {json_chains}')
+            else:
+                    print('exiting, no data exported')
+        else:
+                    print('error occured, exiting')
+
+    if json_rpc_urls:
+        # get data and store it
+        response = requests.get(api_url+ '/all/chains')
+        if response.status_code == 200:
+            data_urls = response.json()
+        else:
+            print(response.text)
+            return
+        # check if path exists
+        if not json_rpc_urls.exists():
+            os.makedirs(os.path.dirname(json_rpc_urls), exist_ok=True)
+            with open(json_rpc_urls, 'w', encoding='utf-8') as f:
+                json.dump(data_urls, f, indent=4)
+                print(f'exported chains to {json_rpc_urls}')
+        elif json_rpc_urls.exists():
+            user_input = input("File already exists for json rpc url, overwrite? (y/n): ")
+            if user_input in ['y', 'Y', 'yes', 'Yes', 'YES']:
+                with open(json_rpc_urls, 'w', encoding='utf-8') as f:
+                    json.dump(data_urls, f, indent=4)
+                    print(f'exported rpc urls to {json_rpc_urls}')
+            else:
+                    print('exiting, no data exported')
+                    return
+        else:
+            print('error occured, exiting')
```

Comment:
Same thing for this one, I don't think we'll ever enter this one so might as well remove it.